Gruntfile.js

```diff
@@ -39,9 +39,9 @@ module.exports = function(grunt) {
     'src/renderers/canvas/sigma.canvas.hovers.def.js',
     'src/renderers/canvas/sigma.canvas.nodes.def.js',
     'src/renderers/canvas/sigma.canvas.edges.def.js',
-    'src/renderers/canvas/sigma.canvas.edges.dotCurve.js',
+    'src/renderers/canvas/sigma.canvas.edges.curve.js',
     'src/renderers/canvas/sigma.canvas.edges.arrow.js',
-    'src/renderers/canvas/sigma.canvas.edges.dotCurvedArrow.js',
+    'src/renderers/canvas/sigma.canvas.edges.dotCurve.js',
```

Comment:
There is no such renderer in the sources. the dotCurve is a plugin if I recall correctly.